charts/sda-svc/values.yaml

```
@@ -156,6 +156,10 @@ global:
     corsMethods: ""
     # @param corsCreds, allow credentials in the request, cors is disabled if false
     corsCreds: false
+    # @param infoText, name of the project
+    infoText: "Example project"
+    # @param inforURL, URL of the project
+    inforURL: " https://example.com/project"
```

```suggestion
    # @param infoText, name of the project
    infoText: "About Federated EGA"
    # @param inforURL, URL of the project
    inforURL: "https://ega-archive.org/about/projects-and-funders/federated-ega/"
```

Either the fields should have proper defaults *or* be empty in the values file and validation done in `auth-deploy.yaml` to ensure that they are not empty.

---

Sure, I will add the default values as you suggested.